debian/vzlogger.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=vzlogger
+After=syslog.target network.target ntp.service
+
+[Service]
+ExecStart=/usr/bin/vzlogger -c /etc/vzlogger.conf
+ExecReload=
```

Comment:
this could/should be filled in according to https://github.com/volkszaehler/vzlogger/pull/458